docs/develop/javascript-libraries.md

```diff
@@ -0,0 +1,139 @@
+---
+title: 'Using built-in JavaScript libraries in Office Scripts'
+description: 'How to call built-in JavaScript APIs from an Office Script in Excel on the web.'
+ms.date: 01/21/2020
+localization_priority: Normal
+---
+
+# Using built-in JavaScript libraries in Office Scripts
+
+JavaScript has several built-in objects any JavaScript code can use. The [TypeScript](../overview/code-editor-environment.md) of Office Scripts is a superset of JavaScript, so it also includes these objects. This article focuses on a few select objects and how they can be used by a script to integrate with your Excel workbook. Mozilla's [Standard built-in objects](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects) contains a complete list of these objects.
+
+## Array
+
+The [Array](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array) object gives your script functions to work with array types. While arrays are standard JavaScript constructs, they relate to Office Scripts in two major ways: ranges and collections.
+
+Ranges contain several two-dimensional arrays that directly map to the cells in that range. These include properties such as `values`, `formulas`, and `numberFormat`. Your script needs to load the related property before reading any cell in that array, for example `myRange.load("values")` (as well as `await context.sync();` to complete the `load` operation).
+
+The following script searches the **A1:D4** range for any number format containing a "$". The script sets the fill color in those cells to "yellow".
+
+```TypeScript
+async function main(context: Excel.RequestContext) {
+  // Get the range From A1 to D4.
+  let range = context.workbook.worksheets.getActiveWorksheet().getRange("A1:D4");
+
+  // Load the numberFormat property on the range.
+  range.load("numberFormat");
+  await context.sync();
+
+  // Iterate over the entire range.
```

Comment:
Consider rephrasing this comment to convey the relevance of arrays. maybe something like ->

// Iterate through the arrays of rows and columns that represent the range.